app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".LoginFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/iv_login_cover"
+            android:layout_width="match_parent"
+            android:layout_height="200dp"
+            app:layout_constraintTop_toTopOf="parent"
+            android:src="@drawable/login_cover" />
+
+        <ImageView
+            android:id="@+id/iv_wolox_logo"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="@id/iv_login_cover"
+            app:layout_constraintTop_toBottomOf="@id/iv_login_cover"
+            android:src="@drawable/wolox_logo"/>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            app:layout_constraintTop_toBottomOf="@id/iv_wolox_logo"
+            android:padding="@dimen/padding">
+
+            <EditText
+                android:id="@+id/etMail"
```

Comment:
Let's change this ID to `edit_text_email`. The same applies for the rest of the IDs on this XML, ok?